pkg/injector/injector.go

```diff
@@ -41,6 +41,7 @@ var allowedControllersServiceAccounts = []string{
 	"cronjob-controller",
 	"job-controller",
 	"statefulset-controller",
+	"daemon-set-controller",
```

Comment:
**Question for maintainers:** So what is the canonical source of truth for this list?

Reply:
This is the closest thing I found: https://github.com/kubernetes/website/blob/3ca34b9a3be454055ba234f1d2ff7d55809b5040/content/en/docs/reference/access-authn-authz/rbac.md#roles-for-built-in-controllers-controller-roles